src/ontology/foodon-edit.owl

```diff
@@ -19743,6 +19796,14 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
     
 
 
+    <!-- http://purl.obolibrary.org/obo/FOODON_03302012 -->
+
+    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/FOODON_03302012">
```

Comment:
This isn't correct.  A frankfurter is just the sausage itself.  A frankfurter in a bun can't be a subclass of frankfurter.